### PR TITLE
Go annotation

### PR DESCRIPTION
**Pull request by TomConlin**
addresses #766 

refactor column headers etc
add s couple of new species types(mould & fungi)
bump date for monarch archive
change  

    '<gene> <has_synonym> "prot_curie" .'
to    

     '<gene> <same _as> <prot_curie> .'

(open to alternative associations)

### CODE REVIEW

**Comment by kshefchek** [comment]
> '<gene> <same _as> <prot_curie> .'

I don't mind the simplification but we probably should align each source with this change.  In other sources we use 'has gene product' to link proteins to genes.  So if we half commit to gene-protein equivalence we'll end up with has gene product cycles.

File: dipper/sources/GeneOntology.py [hunk 1]
```diff
@@ -318,46 +387,46 @@ def process_gaf(self, file, limit, id_map=None, eco_map=None):
...
                 # Derive G2P Associations from IMP annotations
                 # in version 2.1 Pipe will indicate 'OR'
                 # and Comma will indicate 'AND'.
                 # in version 2.0, multiple values are separated by pipes
                 # where the pipe has been used to mean 'AND'
                 if eco_symbol == 'IMP' and with_or_from != '':
-                    withitems = re.split(r'\|', with_or_from)
-                    phenotypeid = go_id+'PHENOTYPE'
+                    withitems = with_or_from.split('|')
+                    phenotypeid = go_id + 'PHENOTYPE'
```
Comment (kshefchek) [inline dipper/sources/GeneOntology.py]: I know it's just formatting, but should we be moving away from these? cc @matentzn @cmungall

**Comment by TomConlin** [comment]
I like 'has gene product' much better than 'same_as'  
which a grabbed base on being proximally the closest thing to has_synonym in the model class

File: dipper/sources/GeneOntology.py (hunk 1 above)
Comment (TomConlin) [reply dipper/sources/GeneOntology.py]: +1

**Pull request merged**